### PR TITLE
Merge story 3 into master

### PR DESCRIPTION
**Pull request by hoptond**
Wrote the HTML and CSS for displaying each element of the approved wireframe. Please review index.html and css/game.css and also comment on the graphics if you feel they could do with changes.

Also review our JS stuff that I forgot to put in seperate branch because I'm smart

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Rocket Man</title>
+    <link rel="stylesheet" href="css/game.css" type="text/css">
+</head>
+<body>
+<main>
```
Comment (patrickkenyon) [inline index.html]: Main should be indented.

File: css/game.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.keyboard {
+    display: inline-block;
+    background-color: #000000;
+    color: #71b264;
+    width: 50px;
+    height: 50px;
+    border-radius: 100%;
+    margin: 20px 98px 0 98px;
```
Comment (patrickkenyon) [inline css/game.css]: You can omit the last value as left and right are the same

File: js/anims-lasers.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+function shootLaser($city) {
+    var cityid = $city.data('city')
+    $city.attr('src','img/cities-' + cityid + '-laser.png')
+    var $this = $city
+    setTimeout(function () {
+        $this.attr('src','img/cities-' + cityid + '.png')
+    }, 300)
+}
+$('.city').click(function () {
```
Comment (f-gate) [inline js/anims-lasers.js]: Needs to go inside document ready function.

File: js/anims-lasers.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+function shootLaser($city) {
+    var cityid = $city.data('city')
+    $city.attr('src','img/cities-' + cityid + '-laser.png')
+    var $this = $city
```
Comment (f-gate) [inline js/anims-lasers.js]: any reason setting this variable?

File: index.html (hunk 1 above)
Comment (hoptond) [reply index.html]: fixed in 168b774

**Review by f-gate — CHANGES_REQUESTED** [review]
lasers off centre

File: css/game.css (hunk 2 above)
Comment (hoptond) [reply css/game.css]: You can't because if we just had two values we'd have 20px margin added to the bottom, which isn't what we want

File: js/anims-lasers.js (hunk 3 above)
Comment (hoptond) [reply js/anims-lasers.js]: fixed inside 0207606

File: js/anims-lasers.js (hunk 4 above)
Comment (hoptond) [reply js/anims-lasers.js]: fixed in 184c6d0

File: css/game.css (hunk 2 above)
Comment (patrickkenyon) [reply css/game.css]: Do it again please!

**Review by f-gate — DISMISSED** [review]

**Comment by patrickkenyon** [comment]
Just the margin to change please!

**Review by patrickkenyon — DISMISSED** [review]

File: css/game.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+img.missile {
+    position: relative;
+    width: 25px;
+    display: block;
+    margin: 0 auto;
+}
+
+
```
Comment (mporam) [inline css/game.css]: whitespace

File: js/anims-lasers.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
+$(function() {
+    $('.city').click(function () {
+        shootLaser($(this))
+    })
+})
+
+document.addEventListener('keypress', function(e) {
+    switch (true) {
```
Comment (mporam) [inline js/anims-lasers.js]: look at the code in patrick and buraks branch

File: css/game.css [hunk 7]
```diff
@@ -0,0 +1,90 @@
+* {
+    box-sizing: border-box;
+}
+
+.game-wrapper {
```
Comment (mporam) [inline css/game.css:100]: nothing confining the height of this box to the agreed dimensions

File: js/anims-lasers.js [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+            break
+        case (e.key === 'r'):
+            shootLaser($('#city-4'))
+            break
+    }
+})
+
+function shootLaser($city) {
+    var cityid = $city.data('city')
+    $city.attr('src','img/cities-' + cityid + '-laser.png')
```
Comment (mporam) [inline js/anims-lasers.js]: this could be done with a class change to update the background image rather than changing the img tag, which means you can control the images from CSS and improve your separation of concerns

**Review by mporam — CHANGES_REQUESTED** [review]

File: css/game.css (hunk 5 above)
Comment (hoptond) [reply css/game.css]: resolved in fea804a

File: js/anims-lasers.js (hunk 8 above)
Comment (hoptond) [reply js/anims-lasers.js]: fixed in 8000153

File: css/game.css (hunk 7 above)
Comment (hoptond) [reply css/game.css:100]: fixed in 8000153

File: js/anims-lasers.js (hunk 6 above)
Comment (hoptond) [reply js/anims-lasers.js]: fixed in 128e995

File: css/game.css [hunk 9]
```diff
@@ -22,11 +31,49 @@ img.missile {
...
 
-
-img.city {
+.city {
     width: 183px;
+    height: 600px;
     display: block;
+    position: absolute;
+    top: 0;
+    left: 14%;
```
Comment (mporam) [inline css/game.css]: this is a code smell. let the citys centre themselves using margin then make the missiles centre using left and transform

**Review by mporam — CHANGES_REQUESTED** [review]

File: css/game.css (hunk 9 above)
Comment (hoptond) [reply css/game.css]: 2a20b30

**Review by mporam — DISMISSED** [review]
approved code so far

File: index.html [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+            <div id="splash-title-rocket">
+                <img src="img/fire-png-gif-489.gif" id="splash-flames">
+                <img src="img/rocket-large.png" id="splash-rocket-img">
+            </div>
+        </div>
+        <div id="splash-open-game">Play Me</div>
+        <div id="splash-bottom-wrapper">
+            <img src="img/new-york-large.png" id="splash-city">
+            <div id="splash-instructions">
+                <p>A megalomaniac successfully researched and developed nuclear warheads, he plans to rain down fire and
```
Comment (patrickkenyon) [inline index.html]: A megalomaniac 'has' successfully

File: js/anims-lasers.js [hunk 11]
```diff
@@ -1,8 +1,8 @@
-$(function() {
-    $('.city').click(function () {
-        shootLaser($(this))
-    })
-})
+// $(function() {
```
Comment (patrickkenyon) [inline js/anims-lasers.js]: Does this need to be here?

File: index.html (hunk 10 above)
Comment (ucanfil) [reply index.html]: Done.

File: js/anims-lasers.js (hunk 11 above)
Comment (ucanfil) [reply js/anims-lasers.js]: Removed.

**Review by patrickkenyon — DISMISSED** [review]

File: css/styles.css [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+    display: block;
+    width: 200px;
+    padding: 10px;
+    margin: 50px auto 50px auto;
+    font-size: 3.2rem;
+    text-align: center;
+    border-style: solid;
+    text-decoration: none;
+    color: #ffffff;
+    position: relative;
```
Comment (mporam) [inline css/styles.css]: not clear this is clickable, give it a `cursor: pointer;`

Comment (mporam) [reply css/styles.css]: and maybe a hover state?

File: index.html [hunk 13]
```diff
@@ -4,8 +4,36 @@
...
+            <h1 id="splash-title">ROCKET MAN</h1>
+            <div id="splash-title-rocket">
+                <img src="img/fire-png-gif-489.gif" id="splash-flames">
+                <img src="img/rocket-large.png" id="splash-rocket-img">
+            </div>
+        </div>
+        <div id="splash-open-game">Play Me</div>
+        <div id="splash-bottom-wrapper">
+            <img src="img/new-york-large.png" id="splash-city">
+            <div id="splash-instructions">
```
Comment (mporam) [inline index.html:22]: the instructions fall off the bottom of the box in firefox

File: JS/audio.js [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+            var $city = $('#city-' + keys[e.key])
+            shootLaser($city)
+        }
+    })
+}
+
+var cities = document.querySelectorAll('.city')
+cities.forEach(function(city) {
+    city.addEventListener('click', function() {
+        var param = 'cheer' + city.dataset.city
```
Comment (mporam) [inline JS/audio.js]: param is a meaningless variable name, please make it mean something

File: JS/audio.js [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+        if (e.key in keys) {
+            var sound = 'cheer' + keys[e.key]
+            makeNoise(sound)
+            var $city = $('#city-' + keys[e.key])
+            shootLaser($city)
+        }
+    })
+}
+
+var cities = document.querySelectorAll('.city')
```
Comment (mporam) [inline JS/audio.js]: why is the keypress event inside a function but this one isnt??

File: JS/audio.js (hunk 14 above)
Comment (mporam) [inline JS/audio.js]: if you make the keys in the array 1,2,3,4 you wont need to add cheer to the string.

File: css/styles.css [hunk 16]
```diff
@@ -0,0 +1,85 @@
+* {
```
Comment (mporam) [inline css/styles.css]: why is there two css files? and why d they both set box-sizing: border-box;

File: index.html [hunk 17]
```diff
@@ -0,0 +1,79 @@
...
+            <div class="keyboard">R</div>
+        </div>
+        <div class="score-wrapper">
+            <div class="score totals">
+                <h2>DEFLECTS:</h2>
+                <p id="score">0</p>
+            </div>
+            <div class="toll totals">
+                <h2>DEATH TOLL:</h2>
+                <p id="toll">1,125,423</p>
```
Comment (mporam) [inline index.html]: why is the default 1.1million?

File: index.html [hunk 18]
```diff
@@ -0,0 +1,79 @@
...
+            </div>
+        </div>
+        <div id="start" class="start">START</div>
```
Comment (mporam) [inline index.html]: why is this here? start what? its set to display: none; please remove

File: js/anims-lasers.js [hunk 19]
```diff
@@ -0,0 +1,7 @@
+function shootLaser($city) {
```
Comment (mporam) [inline js/anims-lasers.js]: does this really need its own file? a lot of HTTP overhead for a small bit of code

File: index.html [hunk 20]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Rocket Man</title>
+    <link rel="stylesheet" href="css/game.css" type="text/css">
```
Comment (mporam) [inline index.html]: no link to normalize? you have included it in the branch but not linked it?

**Review by mporam — CHANGES_REQUESTED** [review]

File: css/styles.css (hunk 16 above)
Comment (hoptond) [reply css/styles.css]: Fixed in 799c687

File: index.html (hunk 17 above)
Comment (hoptond) [reply index.html]: Fixed in e92a8e8

File: index.html (hunk 20 above)
Comment (hoptond) [reply index.html]: Fixed in 7d7f458

File: css/styles.css (hunk 12 above)
Comment (hoptond) [reply css/styles.css]: Fixed in eda5d1e

File: js/anims-lasers.js (hunk 19 above)
Comment (hoptond) [reply js/anims-lasers.js]: fixed in 1c35b68

File: index.html (hunk 18 above)
Comment (hoptond) [reply index.html]: fixed in de56b2c

File: index.html (hunk 13 above)
Comment (hoptond) [reply index.html:22]: cannot reproduce

File: JS/audio.js (hunk 14 above)
Comment (hoptond) [reply JS/audio.js]: changed in 50aae2a

File: JS/audio.js (hunk 15 above)
Comment (hoptond) [reply JS/audio.js]: changed in 531e7e1

File: JS/audio.js (hunk 14 above)
Comment (hoptond) [reply JS/audio.js]: Eventually we will have more sounds in the makeNoise() function so it helps to keep the prefix. We also might (and probably will) replace the cheering with a laser noise since that would be more fitting

Comment (hoptond) [reply JS/audio.js]: see
https://github.com/Mayden-Academy/2018-rocket-man/blob/story-4-miss-note/JS/audio.js

File: JS/audio.js [hunk 21]
```diff
@@ -15,15 +15,25 @@ function listenKeypress() {
...
+function shootLaser($city) {
+    var cityid = $city.data('city')
+    $city.addClass('laser')
+    setTimeout(function () {
+        $city.removeClass('laser')
+    }, 300)
+}
+function listenCityClick() {
+    document.querySelectorAll('.city').forEach(function(city) {
+        console.log('city listen')
```
Comment (mporam) [inline JS/audio.js]: debug code left in

**Review by mporam — CHANGES_REQUESTED** [review]

File: JS/audio.js (hunk 21 above)
Comment (hoptond) [reply JS/audio.js]: removed in f829529

**Review by mporam — APPROVED** [review]
